format.go

```diff
@@ -21,6 +21,21 @@ var (
 	separator = red(" | ")
 )
 
+func disable_colors(v bool) {
```

Comment:
Snake case? Why?

Also can we make it not taking an argument and call it only when necessary?

Reply:
Oh wow! I don't know where the snake case came. Brain fart. I'll fix that. Boolean parameter was primarily for `force color output`: https://github.com/giantswarm/luigi/blob/master/main.go#L14 and secondarily having possibility to write tests verifying output coloring (but these don't exist atm.).

I was a bit puzzled about the need for that forced color output when therer's a switch for no colors, but I didn't change that in these PRs. If you think that can be simplified, I can remove the boolean as well. WDYT?

Reply:
The thing is it should detect `TTY` not being set, then forcing color is useful. For the test sake, we can just call `disableColors()`. But this isn't very important for me. If you find it better how it is, let's keep it.

Reply:
Oh, now that you mention this `TTY` detection, when forcing color is useful, it's actually necessary to explicitly call `disableColors(flag)` because initially I hit ordering bug with variable initialization. So we should not rely on correct ordering of events regarding variable initialization but explicitly re-set them after parsing command line flags. Therefore I'll keep this.